reactor-netty-core/src/main/java/reactor/netty/ByteBufFlux.java

```diff
@@ -283,8 +283,8 @@ public final ByteBufMono aggregate() {
 		                                   output.addComponents(true, list);
 		                               }
 		                               catch (IllegalReferenceCountException e) {
-		                                   if (log.isDebugEnabled()) {
-		                                       log.debug("", e);
```

Comment:
@violetagg should we open an issue or mark this spot for a possible improvement? empty log message doesn't seem ideal.

Reply:
@simonbasle We deliberately do not use https://github.com/netty/netty/blob/21fb48e4d22b5d030b165200e663c11f545f5576/common/src/main/java/io/netty/util/ReferenceCountUtil.java#L107-L120, this use case can happen if we have async cancel and the buffer has been already released.

Reply:
good to know 👍